src/inspector/processing/stats-processing.service.ts

```diff
@@ -102,34 +109,125 @@ export class StatsProcessingService {
     });
 
     // Validator Block Attestation
-    const missAttestationsLastEpoch = await this.storage.getValidatorCountWithMissedAttestationsLastNEpoch(slot, 1);
-    const missAttestationsLastNEpoch = await this.storage.getValidatorCountWithMissedAttestationsLastNEpoch(
+    const missAttestationsLastEpoch = await this.storage.getValidatorCountByConditionAttestationsLastNEpoch(slot, 1, 'attested = 0');
+    const highIncDelayAttestationsLastEpoch = await this.storage.getValidatorCountByConditionAttestationsLastNEpoch(
```

Comment:
May be it's better to make multiple specific functions instead of generic one?
The reason is breaking of abstraction by providing low level conditions.

Reply:
done